docs/source/changelog.rst

```diff
@@ -1,6 +1,37 @@
 Changelog
 =========
 
+0.15.0 / 2017-06-09
+-------------------
+
+Array
++++++
+
+-  Add dask.stats module (:pr:`2269`)
+-  Support ``ufunc.outer`` (:pr:`2345`)
+-  Optimize fancing indexing by reducing graph overhead (:pr:`2333`) (:pr:`2394`)
+-  Faster array tokenization using alternative hashes (:pr:`2377`)
```

Comment:
Also we now support all functions in `numpy.fft`. Some like `fftshift` have been requested by others before. So probably worth noting.